docs/pages/machine-id/guides/databases.mdx

```diff
@@ -0,0 +1,289 @@
+---
+title: Machine ID with Databases
+description: Protecting your databases with Machine ID
+---
+
+In this guide, we will demonstrate how to use Machine ID to access a database
+protected by Teleport from a custom application.
+
+With Machine ID, Teleport issues short-lived certificates, tied to a machine identity,
+that can be rotated, audited, and managed with the same access controls that 
+Teleport provides for human users.
+
+<Figure align="left" bordered caption="Machine ID and Database Access Deployment">
+  ![Machine ID and Database Access Deployment](../../../img/machine-id/machine-id-database-access.svg)
+</Figure>
+
+## Prerequisites
+
+<ScopedBlock scope={["cloud"]}>
+You will need Teleport Cloud 9.3.0 or later.
+</ScopedBlock>
+<ScopedBlock scope={["oss","enterprise"]}>
+You will need to be running Teleport Proxy and Auth Services, version 9.3.0 or
+later.
+</ScopedBlock>
+
+If you have not already put your database behind Teleport Database Access,
+follow the [Database Access Getting Started Guide](../../database-access/getting-started.mdx).
+Database Access supports databases like PostgreSQL, MongoDB, Redis, and much
+more. See our [Database Access Guides](../../database-access/guides.mdx) for
+a complete list.
+
+If you have not already setup Machine ID, follow the
+[Machine ID Getting Started Guide](../getting-started.mdx)
+to familiarize yourself with Machine ID.
+
+Lastly, ensure both the `tbot` and `tsh` executables are available on your
+desired client system.
```

Comment:
I think this sentence should be a bit more explicit that it's referring to your application's host, rather than your local machine.